internal/ir/xds.go

```diff
@@ -1452,9 +1454,12 @@ type DestinationSetting struct {
 	AddressType *DestinationAddressType `json:"addressType,omitempty" yaml:"addressType,omitempty"`
 	// IPFamily specifies the IP family (IPv4 or IPv6) to use for this destination's endpoints.
 	// This is derived from the backend service and endpoint slice information.
-	IPFamily *egv1a1.IPFamily    `json:"ipFamily,omitempty" yaml:"ipFamily,omitempty"`
-	TLS      *TLSUpstreamConfig  `json:"tls,omitempty" yaml:"tls,omitempty"`
-	Filters  *DestinationFilters `json:"filters,omitempty" yaml:"filters,omitempty"`
+	IPFamily *egv1a1.IPFamily `json:"ipFamily,omitempty" yaml:"ipFamily,omitempty"`
+	// DNSLookupFamily specifies a preferred resolution strategy for ip.
+	// This is derived from the backend service and endpoint slice information.
+	DNSLookupFamily *egv1a1.DNSLookupFamily `json:"DNSLookupFamily,omitempty" yaml:"ipFamily,omitempty"`
```

Comment:
We usually create another IR-level struct. It's kind of meaningless here, as the translation is 1:1, but would still be nice for consistency.

Reply:
Right, but is not is the same for IPFamily?